osf/models/analytics.py

```diff
@@ -142,3 +147,17 @@ def get_basic_counters(cls, page):
             return (counter.unique, counter.total)
         except cls.DoesNotExist:
             return (None, None)
+
+    @classmethod
+    def set_basic_counters(cls, page, count, date=None):
+        cleaned_page = cls.clean_page(page)
+
+        if not date:
+            date = timezone.now()
+        date_string = date.strftime('%Y/%m/%d')
```

Comment:
Is this pulled from somewhere else? I suspect this should really be UTC, and at the very least would need a timezone. Unless there's a really good reason not to, let's make this UTC.

Reply:
Make it this way just to make it consistent with the date we used to update those counts. https://github.com/CenterForOpenScience/osf.io/pull/7693/files#diff-4519792f13c44cf51a8eec0288fa674aL77

Reply:
I would see if you can update both of those to UTC without breaking anything, or if something breaks, that it's easy to fix. If it's a complicated change, we can re-evaluate, but non-timezoned dates are a bad idea.

Reply:
@brianjgeiger Can you clarify? `date` is already timezone-aware in this case.

Reply:
The line I'm specifically pointing to is the date_string which is just yyyy/mm/dd

Reply:
No, wait, it's not a datetime, it's just a date. Sigh. Ignore me.